src/main/java/org/kuali/rice/core/v2_0/package-info.java

```diff
@@ -1,2 +1,3 @@
-@javax.xml.bind.annotation.XmlSchema(namespace = "http://rice.kuali.org/core/v2_0", elementFormDefault = javax.xml.bind.annotation.XmlNsForm.QUALIFIED)
+@javax.xml.bind.annotation.XmlSchema(namespace = "http://rice.kuali.org/core/v2_0", elementFormDefault = javax.xml.bind.annotation.XmlNsForm.QUALIFIED,
```

Comment:
Are we publishing javadocs for this JAR?  If not, maybe we could eliminate the package-info file.  I'm asking opinion on this one,  I don't have a terribly strong opinion

Reply:
I'm not aware of any Javadoc publishing that we're currently doing for this repository. However, I still need these package-info files in place, so that JAXB can use more consistent namespace prefixes in its XML output (as per the package annotations). Such namespace prefix setup is needed for the XML namespace fix that I had noted.

Reply:
OK, thanks for clarifying